public_html/index.html

```diff
@@ -15,19 +15,15 @@
     <!-- Latest compiled JavaScript -->
     <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
     
+    <link rel="stylesheet" href="main.css">
     <style>
-      body {
-        /* body of document */ 
-        padding-top: 15px;
-      }
-      .img-responsive-vert {
-        /* vertical implementation of bootstrap class img-responsive */
-        width: auto;
-        max-height: 100%;
+      .navbar-collapse {
+        float: right;
```

Comment:
Should this get mved to the css file? 

I'd maybe look at using Bootstrap's `.pull-right` class on the `.navbar` element instead of `float: right;` here as you could potentially have other `.navbar-collapse` elements on the same page